css/src/components/popover.scss

```diff
@@ -0,0 +1,35 @@
+.popover {
+	display: inline-block;
+	position: relative;
+
+	.popover-summary {
+		color: $hyperlink;
+		list-style: none;
+		cursor: pointer;
+	}
+
+	.popover-content {
+		position: absolute;
+		min-width: 150px;
+		margin-top: $spacer-3;
+		padding: $spacer-6;
+		border: 1px solid $border;
```

Comment:
Figma designs don't have a border as far as I can tell.

Reply:
Added it for the contrast.

Reply:
Updated to show the border for the high-contrast theme only